### PR TITLE
Update ru.yaml

### PR DESCRIPTION
**Pull request by userkilled**
add new line for full translate russian language

### CODE REVIEW

File: i18n/ru.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 LocaleCode: "ru"
-Language: "Russian"
```
Comment (zyrouge) [inline i18n/ru.yaml:2]: I prefer all the "language" tag to be in English. Please revert this change.

**Review by zyrouge — CHANGES_REQUESTED** [review]

File: i18n/ru.yaml (hunk 1 above)
Comment (userkilled) [reply i18n/ru.yaml:2]: I'm back tag
